src/jvm/main/org/jetbrains/kotlinx/lincheck/TransformationClassLoader.java

```diff
@@ -104,6 +99,16 @@ private static boolean doNotTransform(String className) {
                className.equals("kotlinx.coroutines.CancelHandlerBase");
     }
 
+    /**
+     * Some API classes cannot be transformed due to the [sun.reflect.CallerSensitive] annotation.
+     */
+    public static boolean isImpossibleToTransformApiClass(String className) {
+        return className.equals("sun.misc.Unsafe") ||
```

Comment:
I thought Lincheck never transforms `sun.*`, `jdk.*`, and `java.lang.*` in any case

Reply:
Yes, but I think it's convenient to show explicitly that they cannot be transformed for the mentioned reason.

Reply:
If it was written as a comment, it would be easy to forget it during some refactoring or enhancement.